specification/testbase/resource-manager/Microsoft.TestBase/preview/2023-11-01-preview/draftpackage.json

```diff
@@ -0,0 +1,1142 @@
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ExtractFileOperationParameters"
+            },
+            "description": "Parameters supplied to the Test Base Draft Package ExtractFile operation."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The 'ExtractFile' operation of Draft Package was accepted. Follow the Location header to poll for final outcome.",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "URL to query for status of the operation.",
+                "type": "string"
+              },
+              "Location": {
+                "description": "URL to query for status of the operation.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DraftPackageExtractFile": {
+            "$ref": "./examples/DraftPackageExtractFile.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.TestBase/testBaseAccounts/{testBaseAccountName}/draftPackages/{draftPackageName}/generateFoldersAndScripts": {
+      "post": {
+        "tags": [
+          "Test Base Draft Package"
+        ],
+        "operationId": "DraftPackages_GenerateFoldersAndScripts",
+        "description": "Generates folders and scripts",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./testbase.json#/parameters/TestBaseAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DraftPackageNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GenerateOperationParameters"
+            },
+            "description": "Parameters supplied to the Test Base Draft Package Generate operation."
+          }
+        ],
+        "responses": {
```

Comment:
>  "responses": { [](http://example.com/codeflow?start=6&length=16)

you still need a 200 or 204 final response code, in case they use the 202 Location header polling pattern